.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,41 @@
+name: Bug Report
+description: 'Use this template for tracking new bugs. Make sure to add the severity in the issue title (Low, Medium or High)title: "BUG: [SEVERITY]: [BUG DESCRIPTION]"'
+title: "BUG: [SEVERITY]: [BUG DESCRIPTION]"
+labels: [bug]
+body:
+  - type: textarea
+    attributes:
+      label: Bug description
+      description: Provide a clear description of the bug you encoutered.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Repro
+      description: List the steps needed to reproduce the bug. Be as clear and detailed as possible.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: Descibe what should be done for this bug to be considered as fixed. 
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Environment
+      description: Any information about your setup that might be relevant towards troubleshooting the bug (e.g. branch, operating system, browser, etc)
```

Comment:
we could offer specific options here depending on the issue, maybe separate it into:
1. frontend-related: what network is it connected to, if running local which branch, if live which url, etc
2. contract-related: all the details about running the chain: sdk version, how is the chain running, which branch of KREAd, etc

Reply:
I can imagine some duplicate info here depending on how hard they go on the repro section, so we could also add a lil disclaimer saying something like "anything that's not cover on Repro section"

Reply:
Very good point. I created two different templates; one for frontend bugs and one for contract bugs and added the respective fields to each one. Let me know if there is some input field that would be useful that I missed.